docs/members-example/MembersExample.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { ReactVaporStore } from '../ReactVaporStore';
+import { MembersCompositeActions } from './actions/MembersActions';
+import { MembersCompositeView } from './views/MembersCompositeView';
+
+export class MembersExample extends React.Component<any, any> {
+
+  constructor(props?: any, context?: any) {
+    super(props, context);
+
+    ReactVaporStore.dispatch(MembersCompositeActions.setMembers([
```

Comment:
Here, WebStorm is highlighting the dispatch method in red saying that the type is wrong, but tsc runs flawlessly. Any clue?